proposals/NNNN-Add-new-SDL-System-Structure-using-Mediation-Application.md

```diff
@@ -0,0 +1,119 @@
+ - Addition of the function to manage the HMI status
+ - Addition of the function how to reacquire information(such as Policy table etc.) when connecting to another HU
+
+###### 3. To SDL Core
+ - There is no changes from current SDL Core.
+
+###### 4. To SDL HMI
+ - Addition of the guideline for development of HU. (Implementation is handled by each OEM.)
+
+The changes above will be provide a separate proposal for each change after this proposal is accepted.
+
+## Potential downsides
+
+This proposal is to provide new SDL system with low performed ECU such as PTW and low-cost vehicles, thus it assumes that it requires costs of management and maintenance.
+However, it is assumed that the management and maintenance costs will be reduced for the following reasons.
+ - It will use SDL Core same as current one.
+ - There is not any management about SDL HMI, only manages the guideline.
+ - In the case of HU with low performed ECU, there is no OS and the programing language is different. Therefore, generalization is not possible.
+ - The App Library will be shared by adding a function to determine if it is a low performed ECU system.
+From the above, the management and maintenance of the Mediation App would be newly generated.
+
+
+## Impact on existing code
+
+###### 1. App library
+ - As described above, a determination function and a communication destination and method to the new SDL system will be added, but the existing code will not be affected because systems that do not meet the determination conditions or do not have the conditions will continue to operate with the existing basic system.
+
+###### 2. Mediation App
+ - It will be used in the new SDL system, so there will be no impact on the existing code.
+
+###### 3. SDL Core
+ - At this time, there will be no impact on existing code as we plan to use the current SDL Core as is.
+
+###### 4. SDL HMI
+ - In the new SDL system, SDL HMI will be implemented by OEM and SDLC will only provide the guidelines, so there will be no impact on the existing code.
+
+###### 5. Policy table
+ - There will be no impact on existing code as it is created for each system and OEM.
+
+###### 6. SHAID
+ - There will be no impact on existing code as the new SDL system will still use SHAID as before.
+
+###### 7. RPC
+ - For both Mobile and HMI API, there will be no additional RPC, but parameters will be added.
+
+###### 8. Protocol
+ - The new SDL system will use Bluetooth Low Energy (BLE) for communication, so there will be no impact on the existing code.
+
+
+## Alternatives considered
+
+An alternative proposal is the lightweight SDL Core proposed in [SDL 0302 - SDL System Structure for Middle/Low-end Class Model of Powered Two Wheeler](https://github.com/smartdevicelink/sdl_evolution/issues/1004).
+However, this proposal is not a viable alternative because it does not describe specific modifications and has many disadvantages as described below. Therefore, there is no alternative proposal at this time.
```

Comment:
```suggestion
However, this proposal is not a viable alternative because it does not describe specific modifications and has many disadvantages as described below. Therefore, there is no viable alternative proposal at this time.
```